apps/web/src/app/[locale]/challenge/_components/comments/comment.tsx

```diff
@@ -391,6 +394,16 @@ function SingleComment({
       {!isEditing && (
         <div className="-mb-1">
           <ExpandableContent content={comment.text} />
+          <div className="text-muted-foreground flex items-center gap-2 whitespace-nowrap text-xs">
+            <span>
+              {hasBeenEdited
```

Comment:
Maybe we should conditionally render the entire div that you've created.

```tsx
{hasBeenEdited && (
  <div className="text-muted-foreground flex items-center gap-2 whitespace-nowrap text-xs">
   {`Last edited at ${new Intl.DateTimeFormat(undefined, {
                    timeStyle: 'short',
                    dateStyle: 'short',
                  }).format(comment.updatedAt)}`}
  </div>
)}
```

Reply:
Would fix this